packages/cursorless-vscode/src/vscodeApi.ts

```diff
@@ -9,12 +9,55 @@ import { env, window, workspace } from "vscode";
  * of the VSCode API, so the mocks won't work.
  */
 export const vscodeApi: VscodeApi = {
-  workspace,
   window,
   env,
+
+  workspace: {
+    getConfiguration,
+    watchConfiguration,
+    onDidChangeConfiguration,
+  },
```

Comment:
replace with top-level `usingConfiguration`, similar to https://github.com/cursorless-dev/cursorless/blob/pokey/keyboard-improvements/packages/cursorless-vscode/src/usingSetting.ts

Reply:
Done. Unfortunately now will lost all improvements in mocking settings that I had originally introduced